adapters/orbidder/orbidder.go

```diff
@@ -0,0 +1,139 @@
+package orbidder
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+type OrbidderAdapter struct {
+	endpoint string
+}
+
+// MakeRequests makes the HTTP requests which should be made to fetch bids from orbidder.
+func (rcv *OrbidderAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var errs []error
+	var validImps []openrtb.Imp
+
+	// check if imps exists, if not do not return error and do send request to orbidder
```

Comment:
Comment says "do not return error" when it does return an error.

Reply:
fixed to: check if imps exists, if not return error and do send request to orbidder